docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management-Transition.rst

```diff
@@ -1,8 +1,24 @@
 Object Lifecycle Management: Transition
 =======================================
 
+Object lifecycle transition policies enable you to change the location of an
+object or object type based on its age.
+
+Transition policies do not fully transition non-versioned sources. For such 
```

Comment:
The latest news is that for nonversioned source, instead of deleting the object, we will add a tag telling that the object is deleted. (there will still be a tiny chance that we might tag an updated version of the object instead of the one already transitioned, but at least we would not inadvertently delete a more recent object in such case).

So this is still true, but could be extended when we have the details of what we will do exactly.